plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -82,8 +82,16 @@ const writeKey = async (name: string, key: string) => {
     await chmod(keyFile, '600');
 };
 
-const showWarning = async () => {
-    theia.window.showWarningMessage('Che Git plugin can leverage the generated keys now. To make them available in every workspace containers please restart your workspace.');
+const warning = async (message: string) => {
+    theia.window.showWarningMessage(message);
+};
+
+const showReadyToUse = async () => {
+    warning('Che Git plugin can leverage the generated keys now. To make them available in every workspace containers please restart your workspace.');
```

Comment:
Why do we need to wrap one function to another? I think it would be simpler to define message constants and use only one function.

Reply:
yeah removed the middle man